package.json

```diff
@@ -49,6 +51,8 @@
     ]
   },
   "devDependencies": {
+    "@types/classnames": "^2.2.10",
+    "@types/prop-types": "^15.7.3",
```

Comment:
should the types be in `devDependencies` or `dependencies`?

Reply:
So I looked at the [this site](https://fettblog.eu/typescript-react/prop-types/) and it looks like the `@types/prop-types` goes in the devDependencies. I will change the `@types/classnames` to be a dependency although I think we're ok having it be a devDependency. Couldn't find much documentation online about where these should go.

Reply:
As long as we make them consistent, either one is fine with me.